sdk/identity/azure-identity/HISTORY.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-### 1.1.0b1 Unreleased
+### 1.1.0 Unreleased
```

Comment:
You should remove "Unreleased" yeah?

Reply:
Yes, just waiting until I know the release date 😉